Fixed links in rst files (#323)
ansys/pyansys-dev-guide: doc/source/how-to/dns-configuration.rst

``https://<product>.docs.pyansys.com``

> To request a CNAME for the ``docs.pyansys.com`` domain, contact the

Reviewer: The domain is ``pyansys.com`` and ``docs.pyansys.com`` is a subdomain. Although we usually create ``*.docs.pyansys.com``, let's leave the ``pyansys.com`` reference in here.

```suggestion
To request a CNAME for the ``pyansys.com`` domain, contact the
```

DNS TXT verification
--------------------

> Once a CNAME is registered under the ``docs.pyansys.com`` domain, the next step is

Reviewer: ```suggestion
Once a CNAME is registered under the ``pyansys.com`` domain, the next step is
```

to perform a DNS TXT verification. All PyAnsys subdomains are required by Ansys'
IT department to provide a DNS TXT verification. To verify a new CNAME for an
organization, refer to `Verifying a domain for your organization site`_. This guide
shows how to create DNS TXT verification elements for GitHub Pages sites.

.. warning::

>     Only users with privilege access to the ``docs.pyansys.com`` DNS zone can

Reviewer: ```suggestion
    Only users with privilege access to the ``pyansys.com`` DNS zone can
```

    perform this operation. Contact the `PyAnsys Core Team`_ if needed.

PyAnsys verified domains
------------------------

> In the `PyAnsys GitHub organization`_, this domain has been verified:

Reviewer: ```suggestion
In the `PyAnsys GitHub organization`_, these domains have been verified:
```

> * ``docs.pyansys.com``

Reviewer: ```suggestion
* ``pyansys.com``
* ``docs.pyansys.com``
```
